application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -126,13 +126,12 @@ public void onMessageContext(MessageContextInteractionEvent event) {
 
     @Override
     public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
-        event.deferEdit().queue();
+        event.deferReply(true).queue();
```

Comment:
not sure i need defer here but added it just in case, coz the command was taking some time